.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,10 +22,14 @@ Fixes # (issue)
 <!-- For CHANGELOG separate each item in unreleased section by a blank line to reduce collisions -->
 
 ## PR review
- - [ ] Is this pull request ready for review? (if not, please submit in draft mode)
+Anyone in the community is free to review the PR once the tests have passed.
+Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In in short, see following bullet-list:  
 
-Anyone in the community is free to review the PR once the tests have passed.    
-If we didn't discuss your PR in Github issues there's a high chance it will not be merged.
+ - [ ] Is this pull request ready for review? (if not, please submit in draft mode)
+ - [ ] Check that all items from **Before submitting** are resolved
+ - [ ] Make sure the title is self explanatory and the description concisely explains the PR
+ - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified
```

Comment:
Can you make this clearer? "Bugfixes should be including in bugfix release milestones (m.f.X), features should be included in (m.X.b) releases".